src/launch.md

```diff
@@ -0,0 +1,31 @@
+# Run
+
+Bagua use a built-in module `bagua.distributed.launch` to launch jobs, which has similar usage with Pytorh `torch.distributed.launch`.
+
+The following shows how to start distributed training on single node and multiple nodes.
```

Comment:
We introduce how to start .... in the following sections

Reply:
fixed